test/preload-hover.test.js

```diff
@@ -8,3 +8,10 @@ beforeEach(() => {
 it('should construct', () => {
   new PreloadHover();
 });
+
+it('Creates link based on value passed from parameter', () => {
```

Comment:
By the test name I would expect a DOM operation to be performed and verified. Since the test verifies the configuration is set properly, a better name might be `it('should apply provided defaultDomScope', () => { });` or similar.